src/components/Navbar.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react'
+import { Link } from 'react-scroll'
```

Comment:
This isn't even using link from the router. Remove the router if you are not going to use it. The user will be loading in a whole react packet for no reason.